Cargo.toml

```diff
@@ -106,6 +113,7 @@ rand = { version = "0.8.5", default-features = false, features = ["small_rng"] }
 rustversion = "1.0"
 static_assertions = "1.1"
 testutil = { path = "testutil" }
+serde_json = "1.0.107"
```

Comment:
Why is this needed when we already have an optional non-dev dependency?

Reply:
I use [`serde_json`](https://docs.rs/serde_json/latest/serde_json/) in the tests to simplify the serialization/deserialization and avoid creating my own.    
Since this crate is the one showcased in the [`README.md of serde`](https://github.com/serde-rs/serde#serde-in-action) it is probably one of the most used as well (a quick check at crates.io seems to confirm that)